app/mailers/verify_email_applicant_mailer.rb

```diff
@@ -3,7 +3,7 @@
 class VerifyEmailApplicantMailer < ActionMailer::Base
   helper :applicant_verification
 
-  default from: %("MG::Web" <mg-web@samfundet.no>),
+  default from: %("Samfundet" <mg-web@samfundet.no>),
```

Comment:
I think changing to "mg::web"/"mg-web" makes more sense from an optimization standpoint, it will also reduce the complexity and improve readability.